tools/openblas_support.py

```diff
@@ -20,6 +20,21 @@
 RACKSPACE = 'https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com'
 ARCHITECTURES = ['', 'windows', 'darwin', 'arm', 'x86', 'ppc64']
 
+
+def URLFIX(url):
+    """XXX: remove this function when the OpenBLAS64 wheels are on rackspace"""
+    if 'openblas64_' not in url:
+        return url
+    elif 'manylinux' in url:
+        return url.replace(RACKSPACE, "https://raw.githubusercontent.com/pv/openblas-libs/files-linux-x86_64-1")
+    elif 'macosx' in url:
+        return url.replace(RACKSPACE, "https://raw.githubusercontent.com/pv/openblas-libs/files-osx-x86_64-1")
+    elif 'win_amd64' in url:
+        return url.replace(RACKSPACE, "https://ci.appveyor.com/api/buildjobs/caanscj0958w27kp/artifacts/")
+    else:
+        raise RuntimeError("No uploaded OpenBLAS64_ for this arch!")
+
+
```

Comment:
How did you build the 64-bit artifacts? Would you be interested in making a PR to produce these as part of the regular openblas-libs CI run?

Reply:
It's in: https://github.com/MacPython/openblas-libs/pull/8